### PR TITLE
Jira 1003: navigation test

### PR DESCRIPTION
**Pull request by anastasiyakirylava**
@AnthonyL22 @sddgraham

### CODE REVIEW

File: src/test/java/com/airline/southwest/automation/tests/web/navigation/MainAreasNavigationTest.java [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+        setDriver();
+        driver = new ChromeDriver();
+        wait = new WebDriverWait(driver, 50);
+        driver.get("https://www.southwest.com/");
+        driver.manage().window().maximize();
+    }
+
+    @AfterTest
+    public void afterMethod() {
```
Comment (sddgraham) [inline src/test/java/com/airline/southwest/automation/tests/web/navigation/MainAreasNavigationTest.java]: OMG WHAT WERE YOU THINKING ... Less Monolithic! My Head hurts reading this mess!

File: src/test/java/com/airline/southwest/automation/tests/web/navigation/MainAreasNavigationTest.java [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+        driver.close();
+    }
+
+    @Test(retryAnalyzer = Retry.class, groups = {Groups.ACCEPTANCE_TEST})
+    public void testMainAreasNavigation() {
+
```
Comment (AnthonyL22) [inline src/test/java/com/airline/southwest/automation/tests/web/navigation/MainAreasNavigationTest.java]: Looks okay

File: src/test/java/com/airline/southwest/automation/tests/web/navigation/MainAreasNavigationTest.java [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+        boolean departField = driver.findElement(By.xpath("//input[@id='air-city-arrival']")).isDisplayed();
+        Assert.assertEquals(departField, true);
+
+        driver.findElement(By.xpath("//a[@id='booking-form--hotel-tab']")).click();
+
+        driver.findElement(By.xpath("//a[@id='booking-form--car-tab']")).click();
+        wait.until(ExpectedConditions.visibilityOfElementLocated(By.xpath("//input[@id='car-location-pick-up']")));
+        boolean pickUpLocation = driver.findElement(By.xpath("//input[@id='car-location-pick-up']")).isDisplayed();
+        Assert.assertEquals(pickUpLocation, true);
+
```
Comment (sddgraham) [inline src/test/java/com/airline/southwest/automation/tests/web/navigation/MainAreasNavigationTest.java]: SPACE SPACE SPACE SPACES!

Comment (anastasiyakirylava) [reply src/test/java/com/airline/southwest/automation/tests/web/navigation/MainAreasNavigationTest.java]: i like spaces

**Pull request merged**